docs/reference/triggers.md

```diff
@@ -75,6 +75,12 @@ Trigger your functions from events within AWS by publishing events to various AW
 
 [Read the documentation](https://docs.openfaas.com/openfaas-pro/sqs-events/)
 
+### AWS SNS (OpenFaaS Pro)
+
+Trigger OpenFaaS functions from AWS SNS subscriptions.
```

Comment:
Trigger OpenFaaS functions based upon many different types of events generated in AWS using an AWS SNS subscription.